torch/csrc/autograd/variable.h

```diff
@@ -98,7 +98,7 @@ struct TORCH_API Variable : public at::Tensor {
   /// See NOTE [ Autograd View Variables ] for details.
   friend Variable make_variable_view(
       Variable base,
-      at::Tensor data,
+      const at::Tensor& data,
```

Comment:
Pass by const-reference even for mutable parameters is the outstanding convention in the codebase. @pbelevich even wrote a patch to switch our inplace operators to accept `const&` (because people kept getting confused about the mutable reference.) If we're worried about people getting confused let's beef up the docs.